README.md

```diff
@@ -199,17 +193,19 @@ This function resets the state by calling `initialState` with the current proper
 This pseudo-effect is passed as a property by `injectState`:
 
 ```js
-const Component = injectState({ effects, state, resetState }) => (
-  <form onReset={resetState}>
-    // ...
-  </form>
-)
+const Component = withStore(
+  {
+    initialState: () => ({}),
+    effects: {},
+  },
+  ({ effects, state, resetState }) => <form onReset={resetState}>// ...</form>
```

Comment:
I think the destructuring does not help here, maybe use `store.resetState`?

Reply:
why destructuring does not help ? 
```js
this._store = seal({
          effects: wrappedEffects,
          resetState,
          state: stateProxy,
        });

render() {
        return fn(this._store, this.props);
      }
```